faststream/broker/handler.py

```diff
@@ -317,12 +310,6 @@ async def consume(self, msg: MsgType) -> SendableMessage:  # type: ignore[overri
                         await self.close()
                         return None
 
-                    except Exception as e:
```

Comment:
For now this exception hides all Exceptions, but we need to catch them in the CriticalLogMiddleware